workflows/wf_merlin_magic.wdl

```diff
@@ -419,5 +438,14 @@ workflow merlin_magic {
   String? seroba_ariba_serotype = seroba_task.seroba_ariba_serotype
   String? seroba_ariba_identity = seroba_task.seroba_ariba_identity
   File? seroba_details = seroba_task.seroba_details
- }
-}
\ No newline at end of file
+  # Vibrio
+  File? srst2_vibrio_tsv = srst2_vibrio.srst2_tsv
+  File? srst2_vibrio_detailed_tsv = srst2_vibrio.srst2_detailed_tsv
+  String? srst2_vibrio_version = srst2_vibrio.srst2_version
+  String? srst2_vibrio_ctxA = srst2_vibrio.srst2_vibrio_ctxA
+  String? srst2_vibrio_ompW = srst2_vibrio.srst2_vibrio_ompW
+  String? srst2_vibrio_tcpA_ElTor = srst2_vibrio.srst2_vibrio_tcpA_ElTor
+  String? srst2_vibrio_toxR = srst2_vibrio.srst2_vibrio_toxR
+  String? srst2_vibrio_wbeN_O1 = srst2_vibrio.srst2_vibrio_wbeN_O1
```

Comment:
srst2_vibrio_tcpA_ElTor change to srst2_vibrio_biotype

and

srst2_vibrio_wbeN_O1 change to srst2_vibrio_serotype